src/main/java/org/magicdgs/popgenlib/diversity/NucleotideDiversity.java

```diff
@@ -0,0 +1,160 @@
+        final int numberOfSamples = alleleCounts.stream().reduce(0, Integer::sum);
+        final List<Double> freqs = alleleCounts.stream()
+                .mapToDouble(c -> (double) c / (double) numberOfSamples)
+                .boxed().collect(Collectors.toList());
+        return tajimasPi(numberOfSamples, freqs);
+    }
+
+    /**
+     * Computes Watterson's &theta; using the formula 1.4a in
+     * <a href="http://www.sciencedirect.com/science/article/pii/0040580975900209">
+     * Watterson (1975)</a>
+     *
+     * @param numberOfSamples          number of samples used to get the number of segregating
+     *                                 sites.
+     * @param numberOfSegregatingSites number of segregating sites in the sample.
+     */
+    public static double wattersonsTheta(final int numberOfSamples,
+            final int numberOfSegregatingSites) {
+        Verify.validate(numberOfSegregatingSites >= 0,
+                () -> "Number of segregating sites should be a positive integer or 0: "
+                        + numberOfSegregatingSites);
+        Verify.validate(numberOfSamples > 1,
+                () -> "numberOfSamples should be at least 2: " + numberOfSamples);
+        // - S is the number of segregating sites
+        // - a1 is the denominator of Watterson's theta (seee the method)
+        // formula S / a1
+        return numberOfSegregatingSites / wattersonsDenominatorApproximation(numberOfSamples);
+    }
+
+    /**
+     * Gets the denominator of Watterson's &theta;
+     * (formula 3 in <a href="http://www.genetics.org/content/123/3/585">Tajima (1989)</a>), which
+     * is the {@code numberOfSamples}<SUP>th</SUP> -1 Harmonic Number.
+     *
+     * This method returns the exact value for up to 49 samples. Otherwise, it uses the
+     * Euler–Mascheroni constant (&gamma;) and an approximation for the digamma distribution
+     * (&psi;) for computing the n<SUP>th</SUP> Harmonic Number:
+     *
+     * <p>&gamma; + &psi;({@code numberOfSamples})
+     *
+     * @param numberOfSamples number of samples to compute the denominator.
+     *
+     * @throws IllegalArgumentException if the number of samples is lower than 2.
+     * @see Gamma#digamma(double)
+     */
+    @VisibleForTesting
+    static double wattersonsDenominatorApproximation(final int numberOfSamples) {
+        // Defined as sum(1/j) for j=1 to j=n-1; where n is the number of samples
+        // This is actually the (n-1)th Harmonic number (https://en.wikipedia.org/wiki/Harmonic_number)
+        // This could be more efficiently computed using the formula implying the digamma distribution,
+        // which is implemented in an approximated way in commons-math3 (good enough for our purposes).
+        // In common-math3 they only use the fast algorithm if n is >= 49
+        // thus, the limit for switching to the fast algorithm is 50 here,
+        // because it will use the same number of iterations if not
+        if (numberOfSamples < 49) {
```

Comment:
Good catch! The comment is wrong, because it should read "thus, here the limit for switching to the fast algorithm is 49 too" instead of "thus, the limit for switching to the fast algorithm is 50 here".